content/community/gsoc/2021/ideas.html

```diff
@@ -70,67 +72,67 @@ <h4><img src="/images/App_Generic_32.png"/>Updating and Extending WebPositive</h
 -->
 
 
-<h4><img src="/images/App_Generic_32.png"/>Improving Haiku's WebKit2 port</h4>
+<h3><img src="/images/App_Generic_32.png"/>Improving Haiku's WebKit2 port</h3>
 
-<h5>Idea description</h5>
+<h4>Idea description</h4>
 
 <p>Haiku has a native WebKit port which uses the WebKit1 API. This port is not complete and there are several bugs and minor problems which need to be fixed.
 The plan is to replace this with a more up to date port based on the WebKit2 API. Based on the work completed in GSoC 2019, WebKit2 on Haiku is now able to render some web pages.
 The goal of this project is to continue this work, making the WebKit2 port more reliable, render all pages, handle input events, etc, and make it usable for the WebPositive browser.</p>
 
-<h5>Why we need this</h5>
+<h4>Why we need this</h4>
 
 <p>The web browser is an important part of the operating system today. It is difficult to attract and retain users if we don't provide a good web browser.
 When we started our efforts with WebKit, WebKit2 wasn't ready for our use yet, but today it is more mature and it is time to migrate to it.</p>
 
-<h5>Further reading</h5>
+<h4>Further reading</h4>
 
 <ul>
 <li>Skill set: userland development, exploring a large code-base (WebCore)</li>
 <li>Possible mentors/knowledgeable people: PulkoMandy</li>
-<li><a href="https://github.com/haiku/haikuwebkit/pull/4">Sourcecode of GSoC 2019 work</a></li>
+<li><a href="https://github.com/haiku/webkit">Sourcecode (see the webkit2 branch)</a></li>
```

Comment:
You have undone one of my fixes here. https://github.com/haiku/haikuwebkit/pull/4 is the correct URL